website/addons/owncloud/static/owncloudAnonymousLogActionList.json

```diff
@@ -0,0 +1,3 @@
+{
+    "owncloud_folder_selected" : "A user selected a folder for this node"
```

Comment:
See [this comment](https://github.com/CenterForOpenScience/osf.io/pull/6075/files#r73200861) on other log action file.